src/fps/example/runnerConfig.toml

```diff
@@ -0,0 +1,31 @@
+[status]
```

Comment:
Can we put a comment in this file to explain what it's doing? 

Generally speaking it would be great to get this flow added to the README.md file so it's explained somewhere. Folks will need to be prompted to know to edit this file when they want the monorepo integration tests ran against the state of their code. This is an opt in action.

Reply:
Let's remove runnerConfig.toml as discussed offline [related](https://github.com/ethereum-optimism/superchain-ops/pull/505#discussion_r1937947336).